### PR TITLE
chore: Upgrading NPM to v1.2.2_hotfix and v1.2.3

### PR DESCRIPTION
**Pull request by vakalapa**
https://github.com/Azure/azure-container-networking/pull/777

This version contains NPM fix which ignores adding hostnetwork pod IPs to ipsets

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -236,11 +236,11 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+v1.2.2_hotfix
 1.2.1
 1.1.8
 1.1.7
 1.1.5
-1.1.4
```
Comment (paulgmiller) [inline vhdbuilder/packer/install-dependencies.sh:243]: still technically used by subscription 90878020-48fc-45be-90d0-e12eb86ad61c. CAn we remove that? Also should we yank .5 and .7 while we're here usually we cache about 3 versions.

**Review by paulgmiller — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (vakalapa) [reply vhdbuilder/packer/install-dependencies.sh:243]: Internal search on the subid returns zero results, sure i can trim the version list.

**Comment by vakalapa** [comment]
@xuto2 FYI. Can you help me with this PR

**Comment by xuto2** [comment]
@vakalapa , in future, @paulgmiller can merge this kind of change as he's got the permission now

**Pull request merged**